web/index.html

```diff
@@ -34,6 +34,15 @@
   </style>
   <link rel="manifest" href="manifest.json">
   <script>
+    let query = window.location.search;
+    if (query) {
+      let searchParams = new URLSearchParams(query);
```

Comment:
URLSearchParams seems to support a nullable string passed in, so you could potentially just do:

```js
let searchParams = new URLSearchParams(window.location.search);
if (searchParams.has('renderer') {
  ...
}
```

Reply:
Done.